app/_src/gateway/kong-manager/enable.md

```diff
@@ -13,14 +13,11 @@ or hybrid mode), you can enable {{site.base_gateway}}'s graphical user interface
 1. Set the [`KONG_ADMIN_GUI_PATH`](/gateway/{{page.release}}/reference/configuration/#admin_gui_path) and [`KONG_ADMIN_GUI_URL`](/gateway/{{page.release}}/reference/configuration/#admin_gui_url) properties in the ([`kong.conf`](/gateway/{{page.release}}/production/kong-conf/)) configuration file to the DNS or IP address of your system, then restart {{site.base_gateway}} for the setting to take effect. For example:
 
     ```bash
-    echo "-e 'KONG_ADMIN_GUI_PATH=/manager' \
-    'KONG_ADMIN_GUI_URL=http://localhost:8002/manager' \
-    kong reload exit" | docker exec -i KONG_CONTAINER_ID /bin/sh
+    docker exec -i KONG_CONTAINER_ID /bin/sh -c "export KONG_ADMIN_GUI_PATH='/'; export KONG_ADMIN_GUI_URL='http://localhost:8002/manager'; kong reload; exit"
```

Comment:
```suggestion
    docker exec -i <KONG_CONTAINER_ID> /bin/sh -c "export KONG_ADMIN_GUI_PATH='/'; export KONG_ADMIN_GUI_URL='http://localhost:8002/manager'; kong reload; exit"
```
Should we make sure that that container ID is called out as a variable here? Especially in this command, where there are env variables with the exact same formatting.